### PR TITLE
Radio button

### PR DESCRIPTION
**Pull request by apostolidhs**
* Configured jest to execute pure js files using the babel configuration
* Setup jest infrastructure (assisting methods, configuration)
* Replaced the current test files with the test utilities
* Created the radio button component in pure js and typing files

Screenshot 2020-11-18 at 4 16 51 PM
Screenshot 2020-11-18 at 4 17 01 PM
Screenshot 2020-11-18 at 4 16 57 PM
Screenshot 2020-11-18 at 4 16 54 PM

### CODE REVIEW

File: src/components/radio-button/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { Text } from "../typography"
+import { Input, Container, IconContainer, StyledIcon } from "./styled"
+
+const RadioButton = ({ label, checked, disabled, value, children, name, margin, ...rest }) => (
+  <Container disabled={disabled} margin={margin}>
+    <Input type="radio" value={value} name={name} disabled={disabled} checked={checked} {...rest} />
+    <IconContainer
+      borderColor={disabled ? "selected" : "border"}
+      margin={children || label ? [0, 4, 0, 0] : null}
```
Comment (novykh) [inline src/components/radio-button/index.js:10]: Put the array to a constant.

File: src/components/radio-button/styled.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  width: 20px;
+
+  box-sizing: border-box;
+  border: 1px solid ${props => getColor(props.borderColor)(props)};
+  border-radius: 100%;
+
+  ${margin}
+`
+export const StyledIcon = styled(Icon)`
+  fill: ${props => getColor(props.color)(props)};
```
Comment (novykh) [inline src/components/radio-button/styled.js:43]: Can we update the Icon component to accept color and sizes like that? I can see it accepts just `small, med, large` now. Which is not ideal.

**Review by novykh — APPROVED** [review]

File: src/components/filter-box/filter-box.tsx [hunk 3]
```diff
@@ -163,7 +163,7 @@ export const FilterBox = ({
   const config = useMemo(() => ({ ...codeMirrorConfig, ...editorConfig }), [editorConfig])
 
   return (
-    <Container className={className} inline={inline}>
+    <Container className={className} inline={inline} >
```
Comment (novykh) [inline src/components/filter-box/filter-box.tsx:166]: why

**Review by novykh — APPROVED** [review]

File: src/components/radio-button/styled.js (hunk 2 above)
Comment (apostolidhs) [reply src/components/radio-button/styled.js:43]: I prefer to do that in onother PR, we have already introduced many changes here :)

**Pull request merged**